blackbox/stratisd.spec

```diff
@@ -40,9 +40,8 @@ Stratisd test build.  This package should not be used in production
 
 
 %prep
-%setup -q -n %{name}-%{version}
-%patch0 -p1
-
+%autosetup -p1
+tar xvf %{SOURCE2}
```

Comment:
I think '--strip-components=1" is necessary here.

Reply:
Resolved.